app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with(model: Comment, url:
+user_post_comments_path(@user, @post)) do |form| %>
+ 
+  <div>
+    <%= form.label     :text %>
+    <%= form.text_area :text %>
+  </div>  
+  <div>
+    <%= form.submit "Add Comment" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [REQUIRED] Hello @eamador1 , it is good that you have added the new action to the comments controller but you need to create this form in the views/comments/new.html.erb file which accepts input for creating comments just like you did for the Post.

When I wanted to create a comment on the first post, I visited the correct route and got this error.

The error message "CommentsController#new is missing a template for request formats: text/html" indicates that there is no corresponding HTML view template for the new action in your CommentsController.

<img width="1439" alt="Screenshot 2024-01-12 at 03 58 46" src="https://github.com/eamador1/blog-app/assets/58692590/bea3d523-6e14-4aa2-b14e-911c67840a11">

Reply:
@Whoistolu I was not getting that error, but it does make sense. Thank you